cmd/createdkg.go

```diff
@@ -90,6 +90,10 @@ func runCreateDKG(ctx context.Context, conf createDKGConfig) (err error) {
 		}
 	}()
 
+	if conf.OperatorENRs == nil {
+		return errors.New("enrs not present")
```

Comment:
Nit: `No ENRs provided`

(do we lowercase everything?)

Maybe `no enrs provided with the flag --operator-enrs`

And can we  indicate that the `--operator-enrs` flag is required for that command easily? It's currently a bit innocuous in the [list of params](https://docs.obol.tech/docs/dv/charon_cli_reference#creating-the-configuration-for-a-dkg-ceremony)

Reply:
Yeah, errors are lower case since they are wrapped with prefixes as it flows back up the stack.